docs/architecture/smart-rollups.md

```diff
@@ -71,6 +71,26 @@ Each block also contains the following internal messages, which are created by t
 
 Smart Rollup nodes can use these internal messages to know when blocks begin and end.
 
+## Commitments
+
+Some Smart Rollup nodes post commitments to layer 1, which include a hash of the current state of the kernel.
+If any node's commitment is different from the others, the protocol compares them and identifies the correct commitment.
+This process ensures the security of the Smart Rollup by verifying that the nodes are running the kernel faithfully.
+
+Only Smart Rollup nodes running in operator or maintenance mode post these commitments on a regular basis.
```

Comment:
There is also a mode called `accuser` that post a commitment iff it sees that it differs from what was posted.
It's for the users that wan't to make sure operator are faithfull but don't wan't to lock 10k if not needed.

Reply:
Added accuser mode.